### ISSUE TITLE
Copyq tray disappears while typing

### ISSUE THREAD

**Issue by wada3n** [hluk/CopyQ#1506]
**Describe the bug**

There is a strange issue that appeared to me these days, which is when calling the tray and typing some letters, the tray completely disappears and then appears after writing other letters or after moving the mouse cursor.

**Screenshots**
![Trimmed-video](https://user-images.githubusercontent.com/13304792/97778162-9af80080-1b75-11eb-8616-9f5f5f258ad8.gif)

**Version, OS and Environment**
CopyQ Clipboard Manager v3.13.0
Qt: 5.12.8
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: Pop!_OS 20.04 LTS

**Comment by hluk** [comment]
What is your desktop environment and window manager?

**Comment by wada3n** [comment]
DE: GNOME 
WM: Mutter

**Comment by hluk** [comment]
Any chance it's using Wayland instead of X11?

Do you see any errors/warning in logs (`copyq logs` or F12 from the main window)?

**Comment by dianedef** [comment]
who are you xD

2020-11-02 18:31 UTC+01:00, Lukas Holecek <notifications@github.com>:

**Comment by wada3n** [comment]
Hi @hluk,
I have no error message,
And I don't use Wayland at all
Could my use of kvantum engine for QT apps be causing the problem?

**Comment by hluk** [comment]
> Could my use of kvantum engine for QT apps be causing the problem?

Yes, that could be the reason. CopyQ itself is not rendering the menu though Qt or a theme may have problems when the menu updates items on search - this is not very common operation.

Quick test shows that the engine is overriding shortcuts in the menu in KDE. Starting the app with `env QT_STYLE_OVERRIDE=kvantum copyq` and pressing <kbd>s</kdb> shows main window instead of typing "s" to search.

But setting kvantum theme in the `qt5ct` and starting the app with `env QT_QPA_PLATFORMTHEME=qt5ct copyq` works fine.

**Comment by Chealer** [comment]
@wada3n: Does this persist on an up-to-date system?
If so, I don't know Kvantum but is it simple enough to test with it disabled?
